internal/environment/default.go

```diff
@@ -6,6 +6,26 @@
 
 package environment
 
+import (
+	"fmt"
+	"os"
+
+	"github.com/corazawaf/coraza/v3/debuglog"
+)
+
 // HasAccessToFS indicates whether the runtime target environment has access
 // to OS' filesystem or not.
 var HasAccessToFS = true
+
+// IsDirWritable is a helper function to check if the WAF has access to the filesystem
+func IsDirWritable(logger debuglog.Logger, dir string) error {
```

Comment:
Unless I am missing something I think logger isn't needed here.

Reply:
Thanks, it was a leftover